unyt/_unit_lookup_table.py

```diff
@@ -186,7 +207,11 @@
         ("atm", (pascal_per_atm, dimensions.pressure, 0.0, r"\rm{atm}", False)),
         ("hp", (watt_per_horsepower, dimensions.power, 0.0, r"\rm{hp}", False)),
         ("oz", (kg_per_pound / 16.0, dimensions.mass, 0.0, r"\rm{oz}", False)),
-        ("ton", (kg_per_pound * 2000.0, dimensions.mass, 0.0, r"\rm{ton}", False)),
+        ("ton", (kg_per_pound * 2000.0, dimensions.mass, 0.0, r"\rm{US ton}", False)),
```

Comment:
```suggestion
        ("ton_US", (kg_per_pound * 2000.0, dimensions.mass, 0.0, r"\rm{US ton}", False)),
```

Reply:
So `ton` is already here and we weren't thinking about US vs UK when we implemented it (mistake). So I'm not sure how to handle it without breaking back-compat, other than making `ton_US` the unit and `ton` the alias (which I guess works).

Reply:
Ah, right, I didn't realise it was already implemented ! nevermind then